src/main/java/uk/gov/hmcts/reform/civil/launchdarkly/FeatureToggleService.java

```diff
@@ -62,6 +62,10 @@ public boolean isNoticeOfChangeEnabled() {
         return internalClient.boolVariation("notice-of-change", createLDUser().build(), false);
```

Comment:
I think it should be better too keep the same because this class is not just used by our side, there will be other team (Civil-Damages) also use this class. I am concerned that if we changed, we need to test all the journeys (including their side).